### ISSUE TITLE
Can't start The Neverhood Chronicles, "Could not open neverhood.dat!"

### ISSUE THREAD

**Issue by Gagert** [libretro/scummvm#18]
## Description
While I can start Neverhood with the official Windows release of ScummVM, I've never been able to do so with any ScummVM libretro core (I'm using the current updated version of this ScummVM core now).

### Expected behavior
Actually starting the game without an error massage in a command prompt. The game with all its files boots without issues in the official ScummVM Windows releases.

### Actual behavior
After starting this libretro core in RetroArch and selecting The Neverhood Chronicles in the menu interface of this core, I'm greeted with an error message in a black console debug window saying: ERROR: StaticData::load() Could not open neverhood.dat!

### Version/Commit
[You can find this information under Information->Core Information / System Information]
- ScummVM core: ScummVM (v2.6.1.4-7-g1f9f256)
- RetroArch: 1.14.0 64-bit

### Environment information
- Device: i7-12700KF, 32gb DDR4 RAM, GTX 1080
- OS: Windows 11 22H2

### Additional context
I have a similar thread dating a few years ago regarding this game running in ScummVM for RetroArch ([here](https://github.com/libretro-mirrors/scummvm/issues/152)). This may be from a discontinued ScummVM libretro fork before this repository became the current libretro fork of ScummVM.

**Comment by spleen1981** [comment]
Have you followed this [steps](https://docs.libretro.com/library/scummvm/#initial-configuration)?
`neverhood.dat` is included in `Scummvm.zip`, which can be downloaded and installed from within RetroArch (while it is bundled in stand alone ScummVM).

**Comment by Gagert** [comment]
I downloaded the Scummvm.zip file via the Online Updater of RetroArch, the game still got the very same error message.

However, according to the guide you provided a link to, I actually had to locate the extras & theme folder for the ScummVM libretro core to use (after starting the ScummVM libretro core, go to Options then Path to locate those folders), even though it all was found in the default system directory of RetroArch itself. I'm pretty used to all the libretro cores automatically being able to locate the necessary files (bios and such) via the system folder of RetroArch without any user input.

After setting the right directories manually inside the ScummVM libretro core itself, Neverhood actually worked!

Thanks for the reply!
